src/command/restore/restore.c

```diff
@@ -1368,14 +1392,23 @@ restoreSelectiveExpression(Manifest *manifest)
                 if (strLstExists(systemDbIdList, includeDb))
                     THROW(DbInvalidError, "system databases (template0, postgres, etc.) are included by default");
 
+                // Error if the db id is in the exclude list
+                if(strLstExists(excludeDbIdList, includeDb))
```

Comment:
fixed in commit a69c81b, thanks!

Reply:
Cool. Just one thing - and this is not your fault because David and I neglected to put this in the CONTRIBUTING document (I will fix that) - when fixing an issue, providing a reply with the commit is great but the author must not resolve the conversation - that is the job of the reviewer. The reply alerts the reviewer that some action was taken then it is up to the review to confirm and resolve the conversation if it meets the request or to reply if further action is required.

Reply:
Oh, right, sorry. Seems logic indeed. I'll pay attention to that, thanks for letting me know :-)